example/02/main_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+   // "testing"
+   "log"
+	"math/rand"
+	"github.com/axamon/gobrain"
+	"github.com/axamon/gobrain/persist"
+
+
+)
+
+func Testmain() {
+    // set the random seed to 0
+    rand.Seed(0)
+
+    // create the XOR representation patter to train the network
+    patterns := [][][]float64{
+      {{0, 0}, {0}},
+      {{0, 1}, {1}},
+      {{1, 0}, {1}},
+      {{1, 1}, {0}},
+    }
+
+    // instantiate the Feed Forward
+	ff := &gobrain.FeedForward{}
```

Comment:
There is something wrong with the indentation

Reply:
nop, don't see it

Reply:
I guess it was solved in some other commit